dynatrace/configuration_v1/maintenance_windows.py

```diff
@@ -103,115 +82,85 @@ def list(self) -> PaginatedList["MaintenanceWindowStub"]:
         return PaginatedList(MaintenanceWindowStub, self.__http_client, f"/api/config/v1/maintenanceWindows", list_item="values")
 
     def create_schedule(
-        self, recurrence_type: str, start: str, end: str, zone_id: str, recurrence_start_time: Optional[str] = None, recurrence_duration: Optional[int] = None, recurrence_day_of_week: Optional[str] = None, recurrence_day_of_month: Optional[int] = None
+        self,
+        recurrence_type: str,
+        start: str,
+        end: str,
+        zone_id: str,
+        recurrence_start_time: Optional[str] = None,
+        recurrence_duration: Optional[int] = None,
+        recurrence_day_of_week: Optional[str] = None,
+        recurrence_day_of_month: Optional[int] = None,
     ) -> "Schedule":
         """
         Create a schedule to be used when creating a maintenance window.
         """
-        recurrence = Recurrence(recurrence_start_time, recurrence_duration, recurrence_day_of_week, recurrence_day_of_month) if recurrence_type != "ONCE" else None
-
-        return Schedule(recurrence_type, start, end, zone_id, recurrence)
+        recurrence = Recurrence.create(recurrence_start_time, recurrence_duration, recurrence_day_of_week, recurrence_day_of_month) if recurrence_type != "ONCE" else None
+        return Schedule.create(recurrence_type, start, end, zone_id, recurrence)
 
     def create(
-        self, name: str, description: str, window_type: str, suppression: str, schedule: Schedule, id: Optional[str] = None, suppress_synthetic: Optional[bool] = False, scope: Optional[Scope] = None, recurrence: Optional[Recurrence] = None, scope_entities: Optional[list] = None
+        self,
+        name: str,
+        description: str,
+        window_type: str,
+        suppression: str,
+        schedule: Schedule,
+        maintenance_window_id: Optional[str] = None,
```

Comment:
@jimm-with-a-j This might break your code, renamed `id` to `maintenance_window_id` as we don't want to overwrite the python built-in `id`